masterlist.yaml

```
@@ -13555,21 +13555,32 @@ plugins:
         <<: *dirtyPlugin
         itm: 5
         udr: 3
+
   - name: 'Blood&Mud.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/12016' ]
-    tag:
-      - Hair
-      - NpcFaces
     msg:
       - <<: *wildEdits
-        subs: [ 'Record names have been changed, you can use the xEdit script **Restore record names from master** to correct them.' ]
-        condition: 'checksum("Blood&Mud.esp", 23A62955)'
+        subs: [ 'Incorrectly renames various records, use the **Restore record names from master** TES4Edit script to fix them.' ]
+        condition: 'checksum("Blood&Mud.esp", 23A62955) or checksum("Blood&Mud.esp", E9B56954)' # Pre- and post-cleaning
```

I'll make note of the checksums in the commit message, so you can remove the comments.

Also, did you double-check to make sure the post-cleaning checksum is the same with 4.0.3 & 4.0.3f? If it isn't, it'd be best to go with 4.0.3 for the cleaning data as well.

```suggestion
        condition: 'checksum("Blood&Mud.esp", 23A62955) or checksum("Blood&Mud.esp", E9B56954)'
```

---

4.0.3f is the version recommended for cleaning by Elminster. It clearly finds fewer ITMs here, but I'm not sure why.

---

I'll try cleaning with 4.0.3 and comparing the results.

---

@Infernio Yeah, but most people will probably use 4.0.3 to clean it, so, in most cases, we'd effectively only be displaying the message before they clean.

---

I usually post cleaning reports with 4.0.3f anyway, it's just situations like these where the post-cleaning checksum matters that we should probably consider which version most people will be using & how that affects when messages like these are displayed.

---

4.0.3 gives the same result. I have no idea where that other report came from 🤷‍♀️ 
```yaml
LOOT Masterlist Entries

  - name: 'Blood&Mud - 4.0.3.esp'
    dirty:
      - <<: *quickClean
        crc: 0x23A62955
        util: '[TES4Edit v4.0.3](https://www.nexusmods.com/oblivion/mods/11536)'
        itm: 36
        udr: 401
    dirty:
      - <<: *quickClean
        crc: 0x26CACD70
        util: '[TES4Edit v4.0.3](https://www.nexusmods.com/oblivion/mods/11536)'
        itm: 1
    clean:
      - crc: 0xFAC5765D
        util: 'TES4Edit v4.0.3'
  ```

---

Probably an older version, but, since the post-cleaning checksum (`FAC5765D`) is different, we should go with that one.

Also, for plugins that require multiple passes, instead of displaying both passes, just add the ITMs/UDR/NAVs from the later passes to the first pass in the cleaning data.